### PR TITLE
Update documentation for Slack options

### PR DESCRIPTION
**Pull request by lindhe**
My first attempt of using the Slack service failed with `invalid_auth`, so I thought that this documentation would need to be updated so not everyone need to play the guessing game.

### CODE REVIEW

**Comment by lindhe** [comment]
Is the example for `username` correct?

**Comment by lindhe** [comment]
@blakepettersson I rebased and added the sign-off, so you need to approve again.

File: docs/services/slack.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ If you want to send message using incoming webhook, you can use [webhook](./webh
...
-* `username` - optional, the app username
-* `icon` - optional, the app icon, e.g. :robot_face: or https://example.com/image.png
-* `insecureSkipVerify` - optional bool, true or false
+| **Option**           | **Required** | **Type**       | **Description** | **Example** |
+| -------------------- | ------------ | -------------- | --------------- | ----------- |
+| `apiURL`             | False        | `string`       | The server URL. | `https://example.com/api` |
+| `channels`           | False        | `list[string]` |                 | `["my-channel-1", "my-channel-2"]` |
+| `icon`               | False        | `string`       | The app icon.   | `:robot_face:` or `https://example.com/image.png` |
+| `insecureSkipVerify` | False        | `bool`         |                 | `true` |
+| `sigingSecret`       | False        | `string`       |                 | `8f742231b10e8888abcd99yyyzzz85a5` |
```
Comment (blakepettersson) [inline docs/services/slack.md:15]:
```suggestion
| `signingSecret`       | False        | `string`       |                 | `8f742231b10e8888abcd99yyyzzz85a5` |
```

**Comment by blakepettersson** [comment]
@lindhe just one minor issue, also someone with write access to this repo needs to approve (that someone not being me). @zachaller perhaps?

File: docs/services/slack.md (hunk 1 above)
Comment (lindhe) [reply docs/services/slack.md:15]: > 1. pride; arrogance; haughtiness
> 2. renewal or resumption of an old dispute
> 3. break out of a disease

https://en.wiktionary.org/wiki/siging

¯\\\_(ツ)_\/¯

**Review by pasha-codefresh — APPROVED** [review]

**Comment by pasha-codefresh** [comment]
Thanks @blakepettersson for review and @lindhe for contribution

**Pull request merged**